MH4U Dex/DamageZonesCollectionViewController.m

```diff
@@ -0,0 +1,92 @@
+//
+//  DamageZonesCollectionViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/9/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "DamageZonesCollectionViewController.h"
+
+#import <CoreData/CoreData.h>
+
+#import "DamageZoneCell.h"
+#import "DamageZone.h"
+
+@interface DamageZonesCollectionViewController ()
+
+@property (weak, nonatomic) IBOutlet UICollectionViewFlowLayout *flowLayout;
+@property (nonatomic, strong) NSArray *damageZones;
+
+@end
+
+@implementation DamageZonesCollectionViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    self.flowLayout.itemSize = CGSizeMake(CGRectGetWidth(self.collectionView.bounds), 40.0f);
+    NSError *fetchError = nil;
+    NSFetchRequest *fetchRequest = [[NSFetchRequest alloc] initWithEntityName:@"DamageZone"];
+    NSSortDescriptor *sortDescriptor = [NSSortDescriptor sortDescriptorWithKey:@"id" ascending:YES];
+    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"%K == %@", @"monsterName", self.monsterName];
+    [fetchRequest setPredicate:predicate];
+    [fetchRequest setSortDescriptors:@[sortDescriptor]];
+    self.damageZones = [self.managedObjectContext executeFetchRequest:fetchRequest error:&fetchError];
+}
+
+#pragma mark <UICollectionViewDataSource>
+
+- (NSInteger)numberOfSectionsInCollectionView:(UICollectionView *)collectionView
+{
+    return 2;
```

Comment:
Use an `enum` to define constants for your sections as well as a section-count constant.